src/component/footer/footer.css

```diff
@@ -0,0 +1,19 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
+*{
+    margin: 0px;
+    padding: 0px;
+}
+.footer-container{
+    height: 5rem;
+    width: 100%;
+    text-align: center;
+    line-height: 1.5rem;
+    font-size:  1rem;
+    box-shadow: 0px 0px 5px rgb(139, 139, 139);
```

Comment:
use var for colors

Reply:
I think  color with variable name will not work directly in box-shadow

Reply:
You can use variable name with box-shadow as well. It will work

Reply:
> You can use variable name with box-shadow as well. It will work

You can use variable name with box-shadow as well. It will work

Reply:
> I think color with variable name will not work directly in box-shadow

You can use variable name with box-shadow as well. It will work